framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/ClusteredWorkbenchKieServerPersistentScenarioImpl.java

```diff
@@ -104,6 +104,9 @@ protected void deployCustomResource() {
             registerTrustedSecret(server);
         }
 
+        // Workaround for RHPAM-2830
+        kieApp.getSpec().getObjects().getConsole().setReplicas(1);
```

Comment:
What exactly is broken in this scenario? The linked JIRA mainly describes production environment, not authoring HA.

Reply:
RHPAM-2830 affects to all the environments that use more than one business central (see the environment field). I will update the description to make it updated too.

Reply:
Removed